giskard/ml_worker/websocket/utils.py

```diff
@@ -34,12 +34,15 @@
 from giskard.ml_worker.websocket.action import MLWorkerAction
 from giskard.models.base import BaseModel
 from giskard.path_utils import projects_dir
+from giskard.ml_worker.websocket import AbortParams
 
 logger = logging.getLogger(__name__)
 
 
 def parse_action_param(action: MLWorkerAction, params):
     # TODO: Sort by usage frequency from future MixPanel metrics #NOSONAR
+    if action == MLWorkerAction.abort:
+        return AbortParams.model_validate(params)
```

Comment:
Is this compatible with pyantic 1 ? I think it would be better to stay with parse_obj legacy method for now

Reply:
done